src/services/sanity.js

```diff
@@ -839,18 +855,9 @@ export async function fetchRelatedLessons(railContentId, brand) {
 export async function fetchPackAll(railcontentId) {
   //TODO: Implement getPacks
   const query = `*[railcontent_id == ${railcontentId}]{
-        railcontent_id,
-        "id": railcontent_id,
-        title,
-        "image": thumbnail.asset->url,
-        "artist_name": artist->name,
-        artist,
-        difficulty,
-        difficulty_string,
-        web_url_path,
-        published_on
-      } | order(published_on asc)[0...5]`
-  return fetchSanity(query, true);
+    ${getFieldsForContentType('pack')}
+  } | order(published_on asc)`
```

Comment:
Slicing the array at the request level is a small performance improvement.  
```suggestion
  } | order(published_on asc) [0...1]`
```